pages/leaderboard.tsx

```diff
@@ -143,8 +146,9 @@ export default function Leaderboard({ loginContext }: Props) {
               'md:py-5',
               'md:px-4'
             )}
-            buttonLink={!isLoggedIn ? '/signup' : ''}
-          />
+          >
+            {isLoggedIn && <CountdownTimer />}
```

Comment:
Is it important that this only shows up on when logged in? It seems like it could be relevant to non-logged-in users as well -- they could still be testnet participants that haven't logged in yet.

Reply:
This was a discussed consideration but the designs were around showing the sign up button for not-logged-in users and the countdown timer for logged in users. I don't feel super strongly (and we had both early on), but this was done with intention.